in-person-payments-example/src/main/java/com/adyen/ipp/api/InPersonPaymentsController.java

```diff
@@ -0,0 +1,207 @@
+package com.adyen.ipp.api;
+
+import com.adyen.model.nexo.*;
+import com.adyen.ipp.ApplicationProperty;
+import com.adyen.ipp.model.*;
+import com.adyen.ipp.request.*;
+import com.adyen.ipp.response.*;
+import com.adyen.ipp.service.*;
+import com.adyen.ipp.util.IdUtility;
+import com.adyen.service.exception.ApiException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+/**
+ * REST controller for using Adyen checkout API
+ */
+@RestController
+@RequestMapping("/api")
+public class InPersonPaymentsController {
+    private final Logger log = LoggerFactory.getLogger(InPersonPaymentsController.class);
+
+    private final ApplicationProperty applicationProperty;
+
+    @Autowired
+    private PosAbortService posAbortService;
+
+    @Autowired
+    private PosPaymentService posPaymentService;
+
+    @Autowired
+    private PosReversalService posReversalService;
+
+    @Autowired
+    private PosTransactionStatusService posTransactionStatusService;
+
+    @Autowired
+    private TableService tableService;
+
+    private String saleId;
```

Comment:
`saleId` and `poiId` are not  attributes of the controller, but rather global configuration settings: I wouldn't define those as attributes here. The `applicationProperty` can be used directly when it is needed (i.e. parameter when calling `sendReversalRequest`)